hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -45,6 +45,8 @@ message DatanodeDetailsProto {
     optional string networkLocation = 7; // Network topology location
     optional NodeOperationalState persistedOpState = 8; // The Operational state persisted in the datanode.id file
     optional int64 persistedOpStateExpiry = 9; // The seconds after the epoch when the OpState should expire
+    optional int32 initialVersion = 10;  // Initial datanode version. TODO: This is not useful for clients and can be ignored.
```

Comment:
@adoroszlai Ah, I forgot to ask. Do you think we even need to pass `initialVersion` to the client?

It was only used by DN itself (HDDS-4730). Now for HDDS-9752 we only need `currentVersion`, but not `initialVersion`.

Or do we just want to pass both for cleanliness? which is what I have here in this PR.

Reply:
I think we can skip it.